weasyl/test/web/test_interaction.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from weasyl import errorcode
+from weasyl.test import db_utils
+
+
+INVALID_PATHS = (
+    '/note',
+    '/note?noteid=1&noteid=2',
```

Comment:
Again, not really a thing that’s specific to notes. Regression tests are good in general, but extrapolating from this one, we will be testing basic parameter handling on every route.

Reply:
This is fair. I will remove the `INVALID_PATHS`-related tests and keep `test_get_note_permissions`.